active-response/restart.sh

```diff
@@ -34,13 +34,6 @@ PWD=`pwd`
 # Logging the call
 echo "`date` $0 $1 $2 $3 $4 $5" >> ${PWD}/logs/active-responses.log
 
-# Run logtest in managers
-if [ "$TYPE" = "manager" ]; then
-    if !(${PWD}/bin/ossec-logtest -t > /dev/null 2>&1); then
```

Comment:
We need here another way to test everything is OK before restarting the service. We could use `ossec-analysisd -t` as replacement of `ossec-logtest`.